app/scripts/views/mixins/password-mixin.js

```diff
@@ -25,12 +25,21 @@ define(function (require, exports, module) {
 
     setPasswordVisibilityFromButton: function (button) {
       var isVisible = this.$(button).is(':checked');
-      this.setPasswordVisibility(isVisible);
+      var targets = this.getTargets(button);
+      this.setPasswordVisibility(isVisible, targets);
     },
 
-    setPasswordVisibility: function (isVisible) {
+    getTargets: function (button) {
```

Comment:
IMHO `getTargets` is too generic, especially because you use `.password` in it. 
I suggest `getPasswordInputs` or `getAffectedPasswordInputs` or suggest a different name?